textile/chat-features.textile

```diff
@@ -115,6 +115,7 @@ h4(#rooms-lifecycle-operations). Room Lifecycle Operations
 * @(CHA-RL3)@ A room must be explicitly released via the @RELEASE@ operation. This operation is not performed directly on a Room object by the client, but is described here.
 ** @(CHA-RL3a)@ @[Testable]@ If the room is already in the @RELEASED@ status, this operation is no-op.
 ** @(CHA-RL3b)@ @[Testable]@ If the room is in the @DETACHED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
+** @(CHA-RL3m)@ @[Testable]@ If the room is in the @INITIALIZED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
```

Comment:
We already have spec for this CHA-RL3j, we don't need to update this 🤪

Reply:
I've moved the spec points around a bit so the removed one (3i) is at the bottom of the block, to provide better continuity as I'd expect 3j to sit immediately below 3b.